### PR TITLE
fix Issue #100 - Expanded navigation properties MUST be returned, even if they are not specified in $select

### PR DESCRIPTION
**Pull request by dummy3k**
This fixes issue #100.

### CODE REVIEW

File: AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs [hunk 1]
```diff
@@ -442,13 +442,19 @@ public static List<string> GetSelects(this SelectExpandQueryOption clause)
 
         private static List<string> GetSelects(this SelectExpandClause clause)
```
Comment (BlaiseD) [inline AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs:443]: Better to leave this method as it is then remove the [ExpansionIsValid check](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/5ea5f6419aa7535d6bab80b74545af31422beab8/AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs#L526).

**Review by BlaiseD — COMMENTED** [review]
There are also failing tests in the `Web.Tests` project.

Maybe scrap this one and resubmit?

File: AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs (hunk 1 above)
Comment (dummy3k) [reply AutoMapper.AspNetCore.OData.EFCore/LinqExtensions.cs]: Yes this is much cleaner. I also fixed the Web.Tests.

**Review by BlaiseD — APPROVED** [review]

**Comment by BlaiseD** [comment]
Thanks

**Pull request merged**